### PR TITLE
Add get_feature_type -> get_python_type to changelog and migrate script.

### PR DESCRIPTION
**Pull request by coledie**
- [x] Forked repo is synced with upstream -> github shows no code delta outside of the desired.
    https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
- [x] Tests are passing? https://github.com/microsoft/DeepGNN/blob/main/CONTRIBUTING.md#run-tests
- [x] Changelog and documentation updated.
- [x] PR is labeled using the label menu on the right side.

New Behavior
----------------
Add get_feature_type -> get_python_type to changelog and migrate script.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Breaking. Remove FeatureType enum, replace with np.dtype. FeatureType.BINARY -> np.uint8, FeatureType.FLOAT -> np.float32, FeatureType.INT64 -> np.int64.
 
+- Breaking. Rename get_feature_type -> get_python_type.
```
Comment (mortid0) [inline CHANGELOG.md:25]: Wrong section, we already released 0.1.56. It should be in the unreleased section.

File: src/python/deepgnn/migrate/0_1_56.py [hunk 2]
```diff
@@ -34,6 +34,7 @@
             "from numpy import dtype_temp", "import numpy as np"
         )
         raw_output = raw_output.replace("dtype_temp", "np.dtype")
+        raw_output = raw_output.replace("get_feature_type", "get_python_type")
```
Comment (mortid0) [inline src/python/deepgnn/migrate/0_1_56.py:37]: You need to have another migrate script, `0_1_56_1.py`..

File: CHANGELOG.md (hunk 1 above)
Comment (coledie) [reply CHANGELOG.md:25]: No this is an old change that I forgot to document, it is int he 0.1.56.

File: src/python/deepgnn/migrate/0_1_56.py (hunk 2 above)
Comment (coledie) [reply src/python/deepgnn/migrate/0_1_56.py:37]: Same as before

Comment (mortid0) [reply src/python/deepgnn/migrate/0_1_56.py:37]: The problem is this change will not be part of the 0.1.56 package but will be available only in later versions.

**Review by mortid0 — APPROVED** [review]

**Pull request merged**